.github/workflows/ci-pip-workflow.yml

```
@@ -28,67 +28,80 @@ jobs:
             os: ubuntu-latest
             python-version: 3.7
             numpy-pkg: 'numpy'
-            install-type: develop 
+            install-type: develop
             fits-pkg: 'astropy'
             test-data: none
 
           - name: Linux Build (submodule data w/o Matplotlib or Xspec)
             os: ubuntu-latest
             python-version: 3.7
             numpy-pkg: 'numpy'
-            install-type: develop 
+            install-type: develop
             fits-pkg: 'astropy'
             test-data: submodule
 
     steps:
     - name: Checkout Code
       uses: actions/checkout@v2
       with:
-        submodules: 'True'      
+        submodules: 'True'
 
     - name: Pip Testing Setup - Python
       uses: actions/setup-python@v2
       with:
         python-version: ${{ matrix.python-version }}
 
     - name: Pip Testing Setup - Dependencies
+      shell: bash
       env:
         NUMPYVER: ${{ matrix.numpy-pkg }}           #: Required: numpy OR '...'
         FITSBUILD: ${{ matrix.fits-pkg }}                   #: Optional: '' OR '...'
         MATPLOTLIBVER: ${{ matrix.matplotlib-pkg }} #: Optional: '' OR '...'
-      shell: bash
       run: |
         if [ ! -n "${NUMPYVER}" ] ; then
           NUMPYVER='numpy'
         fi
-        pip install ${NUMPYVER} ${FITSBUILD} ${MATPLOTLIBVER} 
-        
-    - name: Build Sherpa
+        pip install ${NUMPYVER} ${FITSBUILD} ${MATPLOTLIBVER}
+
+    - name: Build Sherpa (develop)
+      shell: bash
+      if: matrix.install-type == 'develop'
       run: |
-        python setup.py ${{ matrix.install-type }}
+        pip install -e .[test] --verbose
+
+    - name: Build Sherpa (install)
+      shell: bash
+      if: matrix.install-type == 'install'
+      run: |
+        pip install . --verbose
+
+    - name: freeze
+      shell: bash
+      run: |
+        pip install pytest-xvfb pytest-cov codecov
```

Codecov will also need to be removed here
```suggestion
        pip install pytest-xvfb pytest-cov
```